_posts/2015-05-16-prism-break.md

```diff
@@ -0,0 +1,31 @@
+
+> _Eye of Australian giant cuttlefish (Sepia apama). [Photo](https://www.flickr.com/photos/rling/5101121081/in/photostream/) by Richard Ling / [CC BY-NC-ND 2.0](https://creativecommons.org/licenses/by-nc-nd/2.0/)_
+
+Cuttlefish eyes make use of some interesting properties of light. Specifically, their pupils are unusual W-shaped slits that amplify chromatic aberration, or the way light of different wavelengths bends slightly differently. Chromatic aberration explains purplish fringes sometimes seen in photographs: short wavelengths (blue light) get refracted or bent more than long wavelengths (red light)[^1]. Eliminating these "errors" is hard in any optical system, whether it's eyes or cameras. For the cuttlefish, this aberration signal actually provides valuable color information that is undetectable with only a single photoreceptor type.
+
+Unlike the round human pupil, the squiggly pupil of the cuttlefish lets in light peripherally, off of the optical axis. This means the chromatic effects -- how differently the incoming wavelengths are focused -- are enhanced. Through its normal accommodation (focusing), the cuttlefish can _infer_ the colors of the environment, effectively using the distance it needs to focus as a clue to color.
+
+For this system to work, the environment of the cuttlefish has to be full of cues to help it accommodate, like shadows and texture. Moreover, these features have be fairly sharply segregated spectrally (that is, neighboring objects need to be different and non-overlapping in hue), and fortunately for the cuttlefish they are. With its ersatz color vision mechanism the cuttlefish can't distinguish a broad flat field of a single color, though.
+
+Through computer simulations with different spectral inputs and pupil shapes, the researchers conclude that color information is in principle available to the cuttlefish. This may be the elusive mechanism to explain how cuttlefish make the spectral discriminations that are compatible with their vivid displays and camouflage abilities. New kinds of color vision tests, ones that don't presuppose [opponency](http://en.wikipedia.org/wiki/Opponent_process) or comparison mechanisms, are needed to experimentally determine the color sensitivity of cuttlefish vision; according to the simulations, the cuttlefish eye can distinguish colors unambiguously, as long as objects are at least 0.75 meters away. This kind of analysis of annular pupils also raises the question of how other species, such as certain dolphins, might make use of the cuttlefish's optical trick for color discrimination.
+
+Playing off sensitivity in one domain (color) against another (focal length) is an ingenious adaptation to compensate for apparent sensory deficits. Studying these adaptations, for example the [diversity of pupil shapes in nature](http://jov.arvojournals.org/article.aspx?articleid=2142714), demonstrates that the eye has evolved to be exquisitely tuned to its surroundings.
```

Comment:
Not sure "ingenious" is the appropriate adjective: it seems to imply some "intelligence" at play…